manifests/init.pp

```diff
@@ -111,13 +114,22 @@
       file { $conf_path:
         ensure  => file,
         content => epp('sysstat/sysconfig.epp', {
-          history       => $history,
-          compressafter => $compressafter,
-          sadc_options  => $sadc_options,
-          zip           => $zip,
+          history         => $history,
+          compressafter   => $compressafter,
+          sa_dir          => $sa_dir,
+          sa2_delay_range => $sa2_delay_range,
+          sadc_options    => $sadc_options,
+          sa_umask        => $sa_umask,
+          zip             => $zip,
         }),
       }
 
+      file { $sa_dir:
+        ensure => directory,
+        owner  => 'root',
+        group  => 'root',
+      }
```

Comment:
I think this will work on AIX, but I don't have a system to test on

Reply:
I do not have access to AIX either....

Reply:
oh well, I think it's ok from what I googled, I suppose we'll find out sooner or later if it's not.